libraries/WiFi/src/WiFiMulti.cpp

```diff
@@ -30,6 +30,7 @@
 
 WiFiMulti::WiFiMulti()
 {
+    ipv6_support = false;
```

Comment:
Is this the default?  Why default to turning off half the internet? By default clients should probably support the full IP stack (both); note that if the network only gives an IPv4 address (or only IPv6), then the other half won't be used -- but we don't know in advance what the network will support (or potentially support in the future), so should generally accept "whatever IP the network provides".

Or is it to reduce the size of the binary?  If so, why not also have ipv4_support flag. If you know you are going to be operating in an IPv6 environment (or dual stack), then you could reduce the size by getting rid of IPv4 handling.

Reply:
I can imagine this to be a proper default when the code for IPv6 in this SDK isn't yet fully mature.
Then the ones knowing about IPv6 will turn it on and experiment with it, ironing out all the bugs.